### PR TITLE
fix: create volume for nodes modules and .nuxt / clean up packages

### PR DESCRIPTION
**Pull request by Lucas-Malatchoumy**
# Clean Nuxt stack
- Delete nuxtjs/style-resources package that was blocking hmr
- Create internal volume for node modules
- Create internal volume for .nuxt folder
- Change eslint and prettier config

### CODE REVIEW

File: assets/variables.scss [hunk 1]
```diff
@@ -2,4 +2,5 @@
 //
 // The variables you want to modify
 // $font-size-root: 20px;
-$test: blue
\ No newline at end of file
+$test: blue;
+$toto: red
```
Comment (Maengdok) [inline assets/variables.scss]: Je te le dis maintenant pour éviter l'oublie, mais attention aux noms de tes variables 😜

File: nuxt.config.js [hunk 2]
```diff
@@ -4,35 +4,36 @@ export default {
...
   server: {
-    host: '0.0.0.0',
+    host: '0.0.0.0'
   },
 
   // Global page headers: https://go.nuxtjs.dev/config-head
   head: {
     titleTemplate: '%s - timer',
-    title: 'timer',
+    title: 'toto',
```
Comment (Maengdok) [inline nuxt.config.js]: Attention à ton nommage ici aussi 😂

**Review by Maengdok — CHANGES_REQUESTED** [review]

File: nuxt.config.js (hunk 2 above)
Comment (Lucas-Malatchoumy) [reply nuxt.config.js]: [612cd82](https://github.com/Axel-Pion-MDS/M1_Project_Timer_Front/pull/1/commits/612cd8276f4833fb2ac334e9fab26f66409f4ae4)

File: assets/variables.scss (hunk 1 above)
Comment (Lucas-Malatchoumy) [reply assets/variables.scss]: [612cd82](https://github.com/Axel-Pion-MDS/M1_Project_Timer_Front/pull/1/commits/612cd8276f4833fb2ac334e9fab26f66409f4ae4)

**Review by Maengdok — APPROVED** [review]

**Pull request merged**